samples/setClientEndpoint.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(region = 'us') {
```

Comment:
Looks good content-wise, just needs to be in the same style as other samples. Here's an example: https://github.com/googleapis/nodejs-bigquery/blob/main/samples/clientJSONCredentials.js

Reply:
I thought that the internal function was needed because most samples call BQ API and requires an `async function`.  I have followed the style of this sample that doesn't have the internal function: https://github.com/googleapis/nodejs-bigquery/blob/main/samples/setUserAgent.js